### PR TITLE
fix integrational tests

### PR DESCRIPTION
**Pull request by ezr-ondrej**
@lzap, do you want me to add an issue for this separately, or will you squash it to your test fixes?

### CODE REVIEW

File: test/integration/discovered_hosts_test.rb [hunk 1]
```diff
@@ -64,15 +65,14 @@ class DiscoveredHostsTest < IntegrationTestWithJavascript
 
     test 'and forwards to editing it' do
       create_host
-      assert_equal edit_discovered_host_path(id: discovered_host),
-                   current_path
+      assert_current_path edit_discovered_host_path(id: discovered_host), ignore_query: true
```
Comment (ezr-ondrej) [inline test/integration/discovered_hosts_test.rb:68]: this actually wait for the redirect to happen.

File: test/integration/discovered_hosts_test.rb [hunk 2]
```diff
@@ -147,12 +147,6 @@ def select_host_checkbox(id)
     page.find("#host_ids_#{id}").click
   end
 
-  def select_from(element_id, id)
```
Comment (ezr-ondrej) [inline test/integration/discovered_hosts_test.rb:150]: I switched to the core implementation (`select2` method), which should keep up with the select2 implementation, so it should be more reliable.

File: test/integration/discovered_hosts_test.rb [hunk 3]
```diff
@@ -9,6 +9,7 @@ class DiscoveredHostsTest < IntegrationTestWithJavascript
     assert discovered_notification_blueprint
     discovered_host.save!
     visit discovered_hosts_path
+    page.evaluate_script("document.getElementById('fixedPropertiesSelector-#{discovered_host.id}').classList.remove('fade')")
```
Comment (ezr-ondrej) [inline test/integration/discovered_hosts_test.rb:12]: The modal should not be fading as sometimes the click event misses the button as it is still moving.

**Comment by lzap** [comment]
Yes this fixes it, thanks a bunch! Incorporated the commit into my other PR: https://github.com/theforeman/foreman_discovery/pull/482

**Pull request closed without merging**